stable/kubemonkey/templates/deployment.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: apps/v1beta2
```

Comment:
Can you please use the `apps/v1` API version? This has been available since Kubernetes 1.9.